### PR TITLE
Switch from whenever/cron to sidekiq-scheduler for EDU

### PR DESCRIPTION
**Pull request by markolson**
It should be a little more operationally simple to have sidekiq-scheduler manage the timing out of jobs rather than having a cron task start a rake task that queues a sidekiq job. Even if multiple servers start, they'll all be storing the same schedule in the same redis instance, so we shouldn't have to worry about jobs being run multiple times by multiple cron/rake invocations.

### CODE REVIEW

File: config/sidekiq_scheduler.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+transmit_education_spool_files:
+  cron: "0 3 * * * America/New_York"
+  class: EducationForm::CreateDailySpoolFiles
+  queue: default
```
Comment (lihanli) [inline config/sidekiq_scheduler.yml]: queue is set already in the class, do we need to add it again here?

Comment (aub) [inline config/sidekiq_scheduler.yml]: Also, why call the section transmit_education_spool_files while the job is called CreateDailySpoolFiles?

**Comment by markolson** [comment]
@aub I think there's a place for cron (using `whenever` or some other method to define the scheduling), but in this case it just felt simpler to use sidekiq-scheduler once all was said and done. Plus there's the pretty sidekiq-web UI that we get with this 😉

File: config/sidekiq_scheduler.yml (hunk 1 above)
Comment (markolson) [inline config/sidekiq_scheduler.yml]: No, it's not strictly necessary. I thought it was required for it to show the queue name in the Web UI, but I guess not.

**Comment by aub** [comment]
Yeah, I think there are a number of reasons to prefer this over cron. I'm not sure how to say "this is the official way to do cron-type stuff in the app", but I'd be pro doing so unless there's a really compelling use case for cron.

**Comment by aub** [comment]
Meanwhile, I think this is a fine change, apart from the little nits above.

**Pull request merged**